tests/conftest.py

```diff
@@ -142,7 +142,7 @@ def create(indexer=INDEXER_TVDBV2, indexerid=0, lang='', quality=Quality.UNKNOWN
 def create_tvepisode(monkeypatch):
     def create(show, season, episode, filepath='', **kwargs):
```

Comment:
Most likely, but my first local attempt I did some mass replaces assuming that all `show`s in a file that were named similar and found that wasn't the case.  For these commits so far, I have restricted changes to those logged during actual usage.  I figured questionable usage could come in either later commits or, preferably, later PRs.